packages/devtools_app/release_notes/NEXT_RELEASE_NOTES.md

```diff
@@ -35,7 +35,8 @@ TODO: Remove this section if there are not any general updates.
 
 ## Debugger updates
 
-TODO: Remove this section if there are not any general updates.
+* Added a tooltip to describe the exception mode drop-down. -
+[#4876](https://github.com/flutter/devtools/issues/4876)
```

Comment:
please link this PR instead of the issue that it addressed. Thanks!

Reply:
Sure thing!